### ISSUE TITLE
Show that waiting time of a passenger

### ISSUE THREAD

**Issue by mistotebe** [magwo/elevatorsaga#77]
I've spent ages trying to find out what behaviour of my elevators causes the max wait time to run out on the levels. It is really hard to debug such problems without #34 or having a way to spot the waiting time of each passenger easily. Maybe changing the colour of each passenger from white through blue, green, up to red as a function their current wait time? Would make the game much nicer to look at and maybe even provide more motivation for some people to keep the elevators moving in the minimum moves challenges.

**Comment by magwo** [comment]
Yeah. Not sure how to visualize this without significant clutter. The waiting time is irrelevant for the starting levels. What do you think about some kind of hover tooltip?

**Comment by larva** [comment]
I've got an experimental branch that shows the "Wait" time of each passenger layered on top of them (NOT above their heads as that entails changing floor + elevator layouts and could be easily confused with the floor numbers on elevators..).

In some un-pushed commits I've also added bits to let players switch between showing portions of the total time; breaking it down into time spent waiting for an elevator (renamed "wait time") and time spent on an elevator ("travel time"). I renamed "Wait" time to "Commute" so that Commute = Wait + Travel.

It's not quite ready yet but if anyone wants to review it see https://github.com/larva/elevatorsaga/commits/show-user-times-v2

**Comment by dedavidd** [comment]
I wrote a small patch that changes the color based on the waiting time, probably will tweak a bit

https://github.com/dedavidd/elevatorsaga/commit/a8d07d57260ce7016d90429bfed780450a0ef7e6

**Comment by jenspfahl** [comment]
Let the people wave their hands or render a "Zzzz"-bubble on their heads to indicate long waiting times.

**Comment by avodonosov** [comment]
Interesting that style.css has yellow and red color for disappointed and frustrated users:
https://github.com/magwo/elevatorsaga/blob/e0c55bf8cf59243c32dc371b2cc7b2e3ec1664af/style.css#L245

Although nowhere in the code those styles are set.

Looks like that is a not completed feature.
